### PR TITLE
setup authentication, no code

### PR DESCRIPTION
**Pull request by chienmp**
## Related Tickets:
- [#33585](https://edu-redmine.sun-asterisk.vn/issues/33585)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by chienmp** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
update lại ticket, đặc biệt là description

**Comment by chienmp** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -16,3 +16,7 @@
 Route::get('/', function () {
     return view('welcome');
 });
```
Comment (huyenttm-0901) [inline routes/web.php]: không viết logic trong route

**Comment by chienmp** [comment]
ready

**Review by congnx-2423 — APPROVED** [review]

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**